mscore/pianoroll.cpp

```diff
@@ -205,15 +205,15 @@ PianorollEditor::PianorollEditor(QWidget* parent)
             settings.endGroup();
             }
 
-      QActionGroup* ag = new QActionGroup(this);
-      ag->addAction(getAction("delete"));
-      ag->addAction(getAction("pitch-up"));
-      ag->addAction(getAction("pitch-down"));
-      ag->addAction(getAction("pitch-up-octave"));
-      ag->addAction(getAction("pitch-down-octave"));
-
-      addActions(ag->actions());
-      connect(ag, SIGNAL(triggered(QAction*)), SLOT(cmd(QAction*)));
+      actions << getAction("delete");
+      actions << getAction("pitch-up");
+      actions << getAction("pitch-down");
+      actions << getAction("pitch-up-octave");
+      actions << getAction("pitch-down-octave");
```

Comment:
Best would be to not use << at all but append. I believe it's how it done everywhere.
I would even make an array with the strings and then loop to populate the Qlist with append...